plugins/cpu/fu-cpu-device.c

```diff
@@ -92,37 +100,149 @@ fu_cpu_device_init (FuCpuDevice *self)
 	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_INTERNAL);
 	fu_device_add_icon (FU_DEVICE (self), "computer");
 	fu_device_set_version_format (FU_DEVICE (self), FWUPD_VERSION_FORMAT_HEX);
+	fu_device_set_physical_id (FU_DEVICE (self), "cpu:0");
```

Comment:
This shouldn't be hardcoded in case of a multi-socket system no?

Reply:
I'm explicitly just querying the first physical processor;I don't think it's super interesting to enumerate all of them like before.